tests/std/tests/P0220R1_string_view/test.cpp

```diff
@@ -1244,6 +1244,18 @@ static_assert(!is_constructible_v<string, nullptr_t>, "constructing string from
 static_assert(!is_assignable_v<string&, nullptr_t>, "assigning nullptr_t to string is prohibited");
 #endif // _HAS_CXX23
 
+static_assert(is_trivially_move_constructible_v<string_view>);

```

Comment:
Should we move this next to the `is_trivially_copyable_v` tests far above, and add a comment like `// Similar non-standard guarantees:` ? (We generally note when our tests are going beyond the Standard; this helps make it clear what parts of a test are intentional versus accidental, and may eventually help our tests be portable to libc++.)

https://github.com/microsoft/STL/blob/f75c7f596c7b491168fefb5eff5164ab7ae36867/tests/std/tests/P0220R1_string_view/test.cpp#L72-L75